swagger.yaml

```diff
@@ -0,0 +1,158 @@
+## Reference => https://github.com/DinmaOtutu/RIDE-MY-WAY/
```

Comment:
Good morning, Sorry about that but that file was not supposed to be pushed up. That reference has been removed in my current branch that implements the Swagger UI. It will come with my next PR.